app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,3 @@
 <resources>
     <string name="app_name">SampleSDK</string>
-    <!-- Note: Belongs to Managed Consent Database demo Org-->
-    <string name="transcendConsentUrl">https://transcend-cdn.com/cm/63b35d96-a6db-436f-a1cf-ea93ae4be24e/airgap.js</string>
```

Comment:
Is this now coming directly from the api keys for the sdk?

Reply:
No, the user still have to pass the airgapUrl as part of TranscendCoreConfig.

Actually, we used to ask user to create the following field on string.xml and internally our library used to pull the ag url value from here. But since, config properties increased as we developed, moved all these cofigs under `TranscendCoreConfig` 

User would now create an object with required propeties such as airgapUrl, syncDomains and pass that object to the library something like this
https://github.com/transcend-io/consent-manager-android-sample-sdk/pull/21/files#diff-3c6c7c0b7abb1fed94d7b4551cb333350bb379760f418afe1230ae2e54de61aaR66

Reply:
Ah gotcha